manifests/repo.pp

```diff
@@ -34,7 +34,14 @@
       Foreman::Repos['foreman'] -> Package['centos-release-scl-rh']
     }
 
-    if $facts['os']['release']['major'] == '8' and ($repo == 'nightly' or versioncmp($repo, '2.5') >= 0) {
+    if $facts['os']['release']['major'] == '8' and ($repo == 'nightly' or versioncmp($repo, '3.2') >= 0) {
+      package { 'foreman':
```

Comment:
funny how we nowhere have a `package { 'foreman' }` in other places of the code…

Reply:
That also surprised me.